docs/content/AvatarStack.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: AvatarStack
+---
+
+AvatarStack is used to display more than one Avatar in an inline stack.
+
+AvatarStack
+```jsx live
+ <AvatarStack>
+  <AvatarStack.Item alt="Primer" src="https://avatars.githubusercontent.com/primer"/>
+  <AvatarStack.Item alt="GitHub" src="https://avatars.githubusercontent.com/github"/>
+  <AvatarStack.Item alt="Primer 2" src="https://avatars.githubusercontent.com/primer"/>
+  <AvatarStack.Item alt="GitHub 2" src="https://avatars.githubusercontent.com/github"/>
+ </AvatarStack>
+ ```
+
+## System props
+
+AvatarStack components get `COMMON` system props, AvatarStack.Item components do not get any system props. Read our [System Props](/system-props) doc page for a full list of available props.
+
+
+## AvatarStack Component props
+
+| Name | Type | Default | Description |
+| :- | :- | :-: | :- |
+| alignRight | Boolean | | Creates right aligned AvatarStack |
+
+## AvatarStack.Item Component props
+
+| Name | Type | Default | Description |
+| :- | :- | :-: | :- |
+| alt | String | | Adds accessible alternative text for avatars |
+| src | String | | URL for image to be used in avatar |
```

Comment:
Even though these are just native `img` attributes, I'm listing them here because they are significantly important to be able to render the AvatarStack.Item component